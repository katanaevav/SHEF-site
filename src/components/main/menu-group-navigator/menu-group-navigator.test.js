import React from "react";
import {configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import MenuGroupNavigator from "./menu-group-navigator.jsx";

configure({adapter: new Adapter()});

it('Render Order Navigator', () => {
  const tree = shallow(
    <MenuGroupNavigator
      groupId = {1}
      groupName = {`Первые блюда`}
    />
  );
  expect(tree).toMatchSnapshot();
});
