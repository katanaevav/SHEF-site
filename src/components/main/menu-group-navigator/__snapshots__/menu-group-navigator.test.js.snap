// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Order Navigator 1`] = `ShallowWrapper {}`;
